### PR TITLE
docs: added lineBreakStrategyIOS to text and text-input

### PR DESCRIPTION
**Pull request by bang9**
Update Text docs according to the change from https://github.com/facebook/react-native/pull/31272

Thanks!

### CODE REVIEW

**Comment by bang9** [comment]
@Simek FYI, PR(https://github.com/facebook/react-native/pull/31272) was merged!

File: docs/text.md [hunk 1]
```diff
@@ -659,6 +659,16 @@ It allows the user to select text and to use the native copy and paste functiona
 | -------------------------------------------------------- | ------- |
 | enum(`'auto'`, `'text'`, `'none'`, `'contain'`, `'all'`) | `none`  |
 
+---
+
+### `lineBreakStrategyIOS` <div class="label ios">iOS</div>
+
+Set line break strategy on iOS 14+. possible values are `none`, `standard`, `hangul-word`, `push-out`.
```
Comment (Simek) [inline docs/text.md:666]:
```suggestion
Set line break strategy on iOS 14+. Possible values are `none`, `standard`, `hangul-word`, `push-out`.
```

File: docs/textinput.md [hunk 2]
```diff
@@ -945,6 +945,16 @@ The value to show for the text input. `TextInput` is a controlled component, whi
 | ------ |
 | string |
 
+---
+
+### `lineBreakStrategyIOS` <div class="label ios">iOS</div>
+
+Set line break strategy on iOS 14+. possible values are `none`, `standard`, `hangul-word`, `push-out`.
```
Comment (Simek) [inline docs/textinput.md:952]:
```suggestion
Set line break strategy on iOS 14+. Possible values are `none`, `standard`, `hangul-word` and `push-out`.
```

**Review by Simek — APPROVED** [review]
Thank you for the documentation updates, LGTM! 👍 

I have merged in some small tweaks, so we can ship this before 0.71. 😉

**Pull request merged**